clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/provider/agent/DockerRegistryImageCachingAgent.groovy

```diff
@@ -160,6 +161,16 @@ class DockerRegistryImageCachingAgent implements CachingAgent, AccountAware, Age
           }
         }
 
+        if (credentials.inspectDigests) {
+          try {
+            digest = credentials.client.getConfigDigest(repository, tag)
+            digestContent = credentials.client.getDigestContent(repository, digest)
+          } catch (Exception e) {
+            log.warn("Error retrieving config digest for $tagKey; digest and tag will not be cached: $e.message")
+            return
```

Comment:
Since this is optional, I don't think we should return here, but just log the warning, right?

Reply:
yeah probably better to remote `return`.